scripts/windows/build/Create-ArmBase.ps1

```diff
@@ -0,0 +1,78 @@
+<#
+ # This script creates an ARM docker image as a base for EdgeAgent, EdgeHub, and general module.
+ # then pushes it to the appropriate registries.
+ # It assumes that the caller is logged into registries:
+ # edgebuilds.azurecr.io
+ # edgerelease.azurecr.io
+ # hub.docker.com
+ #>
+ 
+param (
+    [ValidateNotNullOrEmpty()]
+    [ValidateScript( {Test-Path $_ -PathType Container})]
+    [String] $ProjectDirectory,
+    
+    [ValidateNotNullOrEmpty()]
+    [String] $DockerNamespace,
+    
+    [ValidateSet("azureiotedge-module-base", "azureiotedge-hub-base", "azureiotedge-agent-base")]
+    [String] $DockerImageName,
+    
+    [ValidateNotNullOrEmpty()]
+    [String] $DockerImageVersion,
+
+    [Switch] $NoPush
+)
+
+$DockerImageArchitecture = "arm32v7"
+
+<#
+ # Docker file
+ #>
+$DockerFile = "$ProjectDirectory/docker/windows/$DockerImageArchitecture/base/Dockerfile"
+if (-Not (Test-Path $DockerFile -PathType Leaf))
+{
+    throw "$DockerFile not found"
+}
+
+<#
+ # Docker registries
+ #>
+$DockerRegistries = New-Object 'System.Collections.Generic.List[String]'
+$DockerRegistries.Add("edgebuilds.azurecr.io/")
+$DockerRegistries.Add("edgerelease.azurecr.io/")
+
+<#
+ # Docker image tags
+ #>
+$DockerImageTags = New-Object 'System.Collections.Generic.List[String]'
```

Comment:
Same as above...

Reply:
Which comment do you refer to? Docker registries? Please clarify.

Reply:
The previous comment about using `New-Object 'System.Collections.Generic.List[String]'` instead of `@("item1", "item2")`. It looks like you marked the comment as resolved without any code change or comments.

Reply:
Updated docker registries to use array instead; but keep docker image tags as list.

Reply:
Update docker image tag variable as well.